### ISSUE TITLE
prop `show-select` cause style problem

### ISSUE THREAD

**Issue by jsz14897502** [spacebnd/vuetify-data-table-resizable-columns#1]
Thank you very much for your work!
But there seems to be a style problem here. When I use prop `show-select` in `<v-data-table>`, the table seems to be broken like this:
![image](https://user-images.githubusercontent.com/32270871/159907448-cfcb5b51-74ad-4958-b4ec-0b43e82f2cb6.png)
My code is as follows:
```
<template>
  <v-card>
    <v-data-table
      dense
      show-select
      single-select
      :headers="headers"
      :items="desserts"
      item-key="name"
      class="elevation-1"
      v-resizable-columns
    ></v-data-table>
  </v-card>
</template>

<script>
export default {
  data: () => ({
    //just as same as vuetify doc
    desserts: [...],
    headers: [...],
  }),
};
</script>
```

And I can resize the cols and then the table will be normal. but after selecting something, the table went wrong again.
Is this a bug or anything I could do to avoid this problem?
Thanks!

**Comment by spacebnd** [comment]
yes, it looks like a bug. 
I'll try to fix it today and release a new version

**Comment by spacebnd** [comment]
> Thank you very much for your work! But there seems to be a style problem here. When I use prop `show-select` in `<v-data-table>`, the table seems to be broken like this: ![image](https://user-images.githubusercontent.com/32270871/159907448-cfcb5b51-74ad-4958-b4ec-0b43e82f2cb6.png) My code is as follows:
> 
> ```
> <template>
>   <v-card>
>     <v-data-table
>       dense
>       show-select
>       single-select
>       :headers="headers"
>       :items="desserts"
>       item-key="name"
>       class="elevation-1"
>       v-resizable-columns
>     ></v-data-table>
>   </v-card>
> </template>
> 
> <script>
> export default {
>   data: () => ({
>     //just as same as vuetify doc
>     desserts: [...],
>     headers: [...],
>   }),
> };
> </script>
> ```
> 
> And I can resize the cols and then the table will be normal. but after selecting something, the table went wrong again. Is this a bug or anything I could do to avoid this problem? Thanks!

yes, it looks like a bug.
I'll try to fix it today and release a new version

**Comment by spacebnd** [comment]
@jsz14897502 
yes, it looks like a bug.
I'll try to fix it today and release a new version

**Comment by spacebnd** [comment]
@jsz14897502 
check new version [1.0.5](https://github.com/spacebnd/vuetify-data-table-resizable-columns/releases/tag/v1.0.5)

**Comment by jsz14897502** [comment]
Thank you for the quick reply. The style doesn't look like before. But I find if I resize the table, and then select something, some dividers seem have wrong style still, like bellow:
![image](https://user-images.githubusercontent.com/32270871/160221940-06c2da74-6e1f-4f65-a69e-fea3e2854fb0.png)

**Comment by spacebnd** [comment]
@jsz14897502 
check update [v1.0.6](https://github.com/spacebnd/vuetify-data-table-resizable-columns/releases/tag/v1.0.6)

this solution has not been tested with props that change the structure of the data-table. 
please let me know if there are any problems elsewhere

**Comment by jsz14897502** [comment]
It works well this time with prop `show-select`. 

Also, as you said, I tested it in some other cases and the table still have some problems.

the first thing is changing headers:
if I delete n of headers,then there would be n+1 dividers still remian (Click on one of them and they all disappear);
and adding headers also cause problems
I'm not sure if my `redraw` usage is correct, here is my code
```
methods: {
    //1.
    del2OfHeaders() {
      this.headers = [
        {
          text: "Dessert (100g serving)",
          align: "start",
          sortable: false,
          value: "name",
        },
        { text: "Protein (g)", value: "protein" },
        { text: "Iron (%)", value: "iron" },
      ];
      this.option = "redraw";
    },

    //2.
    addHeaders() {
      this.headers = [
        {
          text: "Dessert (100g serving)",
          align: "start",
          sortable: false,
          value: "name",
        },
        { text: "Calories", value: "calories" },
        { text: "Fat (g)", value: "fat" },
        { text: "Carbs (g)", value: "carbs" },
        { text: "Protein (g)", value: "protein" },
        { text: "Iron (%)", value: "iron" },
      ];
      this.option = "redraw";
    },
  },
```

Another problem I found is that  when using prop `show-expand`, the expend icons in last cols seem to be partially covered.

**Comment by spacebnd** [comment]
@jsz14897502 
in your case put `this.option = 'redraw'` inside `$nextTick(fn)` (or `setTimeout(fn, 0)`) function and the redraw will work correctly
`
this.$nextTick(() => {
   this.option = 'redraw'
 })
`

`show-expand` prop will be fixed soon, and i'lll test all other props as well. 
thanks for your feedback

**Comment by spacebnd** [comment]
> `show-expand` prop will be fixed soon

fixed in [v1.0.7](https://github.com/spacebnd/vuetify-data-table-resizable-columns/releases/tag/v1.0.7)
